packages/bratus-app/src/components/ComponentNode/ComponentNode.sc.js

```diff
@@ -32,6 +32,9 @@ export const StyledNode = styled.div`
   border: ${({ isHighlighted }) => {
     return isHighlighted ? '1.5px solid black' : '1px solid black';
   }};
+  box-shadow: ${({ isHighlighted }) => {
+    return isHighlighted ? '0px 0px 16px 2.5px rgba(255,194,25,0.76)' : 'none';
```

Comment:
Are you sure about 255,194,24,0.76? 
If yes, let's do it!

Reply:
I was thinking 255,194,24,0.73